app.py

```diff
@@ -14,32 +14,53 @@
 
 from models import Category, Criterion, Link, Score
 from auth import AuthError, requires_auth
+from factories import build_or_update_category
 
 @app.errorhandler(AuthError)
 def handle_auth_error(ex):
 	response = jsonify(ex.error)
 	response.status_code = ex.status_code
 	return response
 
-@app.route("/categories")
+@app.route("/categories", methods=["GET"])
 def get_categories():
-	try:
-		categories=Category.query.all()
-		return  jsonify([category.serialize() for category in categories])
-	except Exception as e:
-		return(str(e))
```

Comment:
Some amount of error handling is built into Flask. I created issue #57 to make sure someone circles back and develops some error handling standards for our app, but I'm not going to deal with that now.